### PR TITLE
updated to correct csp based avise readme document

### PR DESCRIPTION
**Pull request by ericsysmin**

### CODE REVIEW

**Comment by ericsysmin** [comment]
@grastogi23 @chaitanyaavi can you review

File: README.md [hunk 1]
```diff
@@ -1,24 +1,19 @@
 # avinetworks.avise-csp
 
-[![Ansible Galaxy](https://img.shields.io/badge/galaxy-avinetworks.avise-csp-blue.svg)](https://galaxy.ansible.com/avinetworks/avise-csp/)
```
Comment (grastogi23) [inline README.md]: why removing this?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,19 @@
...
 ## Requirements
 
+- A CSP device
 - `avisdk` python library is required and can be installed by:  
 `pip install avisdk --upgrade`  
 
 ## Role Dependencies
 
 - avinetworks.avisdk
-  - To install these use the following command: `ansible-galaxy install -f avinetworks.avisdk`
```
Comment (grastogi23) [inline README.md]: let us leave this in. most people can't figure out that they need to upgrade the role

File: README.md [hunk 3]
```diff
@@ -36,102 +31,114 @@ Using this module you are able to install the Avi Vantage Service Engine, to you
...
-    - role: avinetworks.docker
     - role: avinetworks.avise-csp
-      se_master_ctl_ip: 10.10.27.101
-      se_auth_token: "{{ se_auth_token }}"
+      se_autoregister: false
+      se_auth_token: "auth token here"
+      se_cloud_name: Default-Cloud
+      se_tenant: admin
+      se_csp_user: admin
+      se_csp_password: password
```
Comment (grastogi23) [inline README.md]: make this parameterized so user knows they need to change this.

File: README.md [hunk 4]
```diff
@@ -36,102 +31,114 @@ Using this module you are able to install the Avi Vantage Service Engine, to you
...
-### Example without Auto-registration
+### CSP Example without Auto-registration
 ```
 
-- hosts: all
+---
+- hosts: csp_devices
+  gather_facts: false
   roles:
-    - role: avinetworks.docker
```
Comment (grastogi23) [inline README.md]: I thought we do have dependency on the docker role?

File: README.md [hunk 5]
```diff
@@ -36,102 +31,114 @@ Using this module you are able to install the Avi Vantage Service Engine, to you
...
 ```
 
-### Example without Auto-registration
+### CSP Example without Auto-registration
 ```
 
-- hosts: all
+---
+- hosts: csp_devices
+  gather_facts: false
```
Comment (grastogi23) [inline README.md]: let us add some more details on what this example achieves. Like this is for bond example etc.

File: README.md (hunk 1 above)
Comment (ericsysmin) [reply README.md]: The file doesn't exist and creates a missing image.

File: README.md (hunk 2 above)
Comment (ericsysmin) [reply README.md]: we did, it's here https://github.com/avinetworks/ansible-role-avise-csp/blob/7c225c89bce396eedde1c272e158ed06b8e39600/README.md which is the full file

File: README.md (hunk 3 above)
Comment (ericsysmin) [reply README.md]: they are, look in "Auto-registration parameters"

Comment (ericsysmin) [reply README.md]: and in CSP Deployment variables

File: README.md (hunk 4 above)
Comment (ericsysmin) [reply README.md]: not for csp

File: README.md (hunk 5 above)
Comment (ericsysmin) [reply README.md]: we do, see the example above License statement

**Comment by ericsysmin** [comment]
This was replaced with a better documented, and tied to a Jira and a valid branch https://github.com/avinetworks/ansible-role-avise-csp/pull/13

**Pull request closed without merging**